src/clamp.d.ts

```diff
@@ -0,0 +1,57 @@
+import type { Color, Find, Mode } from './common';
+
+export declare function clampRgb(color: string): Color;
+export declare function clampRgb<C extends Find<Color, M>, M extends Mode>(
+	color: C
+): C;
+
+export declare function clampChroma(color: string): Color;
+export declare function clampChroma<C extends Color>(color: C): C;
+export declare function clampChroma<
+	C extends Find<Color, M>,
+	M extends Mode = 'lch'
```

Comment:
Does the mode really have `lch` as default?

Reply:
https://github.com/Evercoder/culori/blob/bab59612dee69e81b87639be5a863a352960c496/src/clamp.js#L28